bundles/org.openhab.binding.mail/src/main/java/org/openhab/binding/mail/internal/POP3IMAPHandler.java

```diff
@@ -94,6 +94,7 @@ public void initialize() {
             }
         }
 
+        logger.debug("Configuration details:\r\n{}", config.debugPrint());
```

Comment:
```suggestion
        if(logger.isDebugEnabled()){
            logger.debug("Configuration details:\r\n{}", config.debugPrint());
        }
```

Reply:
This method is very low cost. Why do you feel the if conditional is needed here?